arch/arm/configs/bcm2709_defconfig

```diff
@@ -919,6 +919,7 @@ CONFIG_VIDEO_TW9906=m
 CONFIG_VIDEO_IMX219=m
 CONFIG_VIDEO_OV5647=m
 CONFIG_VIDEO_OV7640=m
+CONFIG_VIDEO_IRS1125=m
```

Comment:
Again, Phil will ask that this has been set in the right place via ```make savedefconfig```. Is this in the right place, or is this a manual edit?

Reply:
Manual edit :) I'll change it...

Reply:
Good, because I'm very strict. ;-)

Reply:
@pelwell I fully understand.

That's strange I end up on the same line. So what I did is:
1. Remove irs1125 entry from bcm2709_defconfig
2. make ARCH=arm CROSS_COMPILE=arm-linux-gnueabihf- bcm2709_defconfig
3. vim .config -> there was a line # CONFIG_VIDEO_IRS1125 is not set
4. changed that line to CONFIG_VIDEO_IRS1125=m
5. make ARCH=arm CROSS_COMPILE=arm-linux-gnueabihf- savedefconfig
6. vim defconfig and search for IRS1125 which is again on line 922:

 920 CONFIG_VIDEO_OV5647=m
 921 CONFIG_VIDEO_OV7640=m
 922 CONFIG_VIDEO_IRS1125=m
 923 CONFIG_VIDEO_MT9V011=m
 924 CONFIG_DRM=m

Is this the workflow I should follow?

Reply:
Yes - there are other ways to manipulate the .config file (see scripts/config), but the savedefconfig is the important part. You either have excellent instincts or it was a lucky guess.

Reply:
I guess it's a sign to play lottery today